rules/ruleset_json_test.go

```diff
@@ -510,7 +511,7 @@ func TestRuleSetJSONSortByMappingNameAsc(t *testing.T) {
 	require.Equal(t, expected, rs)
 }
 
-func testRuleSetJSON(namespace string, mappingRules []MappingRuleJSON,
+func testToRuleSetJSON(namespace string, mappingRules []MappingRuleJSON,
```

Comment:
Should just be `testRuleSetJSON`?

Reply:
Looks like I messed up a find and replaces.